beetsplug/info.py

```diff
@@ -239,4 +240,5 @@ def filter_(data):
 
 
 def identity(val):
+    val.pop('path', None)  # path is fetched from item
```

Comment:
It's a little odd to call this function `identity` when it's not really an identity function. :smiley: I'm going to fiddle around with this a little bit…